server/src/packages/packages.service.ts

```diff
@@ -144,16 +133,16 @@ export class PackagesService {
         formData = await this.fetchPackageForm(firstPackage);
       }
 
+      // re-alias project property
+      const {
+        dcp_project,
+      } = firstPackage;
+
       return {
         ...firstPackage,
         ...formData,
```

Comment:
Yes, there's some inconsistency in how we inject related entities like invoices, forms or documents into projects or packages. I think in the future we can look into moving the injection code out of the package and services and more into their respective services